python/hail/dataset.py

```diff
@@ -2261,9 +2231,9 @@ def lmmreg(self, kinship_vds, y, covariates=[], global_root="global.lmmreg", va_
 
         **Examples**
 
-        Suppose ``assoc.vds`` has a Boolean variant annotation ``va.useInKinship`` and numeric or Boolean sample annotations ``sa.pheno``, ``sa.cov1``, ``sa.cov2``. Then the :py:meth:`.lmmreg` function in
+        Suppose ``vds`` has a Boolean variant annotation ``va.useInKinship`` and numeric or Boolean sample annotations ``sa.pheno``, ``sa.cov1``, ``sa.cov2``. Then the :py:meth:`.lmmreg` function in
```

Comment:
incorrect statement about `vds`: you're reading example_lmmreg.vds, so say something like "the VDS saved at foo/bar/..."